### ISSUE TITLE
Add non-Fx content to FxA benefits page

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7035]
## Description

60% of email referrals to FxA benefits page are in non-Fx browsers. Many came from e.g. an email they opened in Mobile Safari or Chrome on Android. The email said, "Get an Account". When they get to this page, they see a big "Get Firefox" button. This is a broken experience.

We should add content, shown to non-Fx visitors, that helps them access a smaller set of services that are supported by FxA but not dependent on Firefox. As far as I know, these services include...

* Send
* Monitor
* Notes
* Pocket

The content for these services is already substantially produced on e.g. [the Whatsnew 66 page](https://www.mozilla.org/en-US/firefox/66.0/whatsnew/), though not in exactly the right form factor. 

This is a short-term tactic to maximize FxA signups by serving non-Fx users landing on this page. The long-term solution is to rethink how we market FxA and build FxA funnels on the site, but that is *out of scope* for this issue.

[Plan](https://docs.google.com/document/d/1_nTLCcVZ5UcbT60yxdj4KI4OkFKCGVrAR2dVUo862vc/edit?usp=drivesdk)

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] A chrome user landing on /firefox/accounts/ can find a way to make an account, if they want to access the non-Firefox-dependent services we offer.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] We may see a drop in Fx acquisitions as a result

#### Tasks
- [ ] produce UX specifications
- [ ] developer review of UX
- [ ] design for new specifications (unlikely)
- [ ] copy for new specifications (likely)
- [ ] code changes to accomplish UX

**Comment by alexgibson** [comment]
Related issues:

- https://github.com/mozilla/bedrock/issues/6907
- https://github.com/mozilla/bedrock/issues/6789
- https://github.com/mozilla/bedrock/issues/6837

**Comment by hoosteeno** [comment]
Blocked by https://github.com/mozilla/bedrock/issues/7062

**Comment by hoosteeno** [comment]
This is ready for review from 2 perspectives: 
1. Code
2. UX/UI

**Comment by hoosteeno** [comment]
@mrkwvr can you review [the screenshot here](https://user-images.githubusercontent.com/94519/56085445-a56e7000-5e00-11e9-96ce-30dd0096311e.png) in light of the description above from a UI/UX perspective, and offer any suggestions you think make it better? Feel free to mock them up if you wish.

**Comment by LaurenNiolet** [comment]
From our perspective on accounts growth, the layout in the screenshot addresses a primary concern, that users who were interested in an account weren't seeing an accounts CTA when they hit the page. I'll certainly defer to @mrkwvr for more sophisticated analysis.

**Comment by mrkwvr** [comment]
The one recommendation would be to use the ghosted (outlined) buttons for all the benefits/products and the blue button for the CTA at the bottom. Right now there are too many blue buttons. I can mock up if that's helpful

**Comment by hoosteeno** [comment]
> The one recommendation would be to use the ghosted (outlined) buttons for all the benefits/products and the blue button for the CTA at the bottom. Right now there are too many blue buttons. I can mock up if that's helpful

This is done.

**Comment by LaurenNiolet** [comment]
what is that bottom CTA if the user is in Firefox?

**Comment by hoosteeno** [comment]
> what is that bottom CTA if the user is in Firefox?

Sync-state dependent:

<img width="1196" alt="Screen Shot 2019-04-15 at 10 27 27 AM" src="https://user-images.githubusercontent.com/94519/56149144-1f216d80-5f69-11e9-880a-dce925d83fb7.png">

<img width="1348" alt="Screen Shot 2019-04-15 at 10 27 10 AM" src="https://user-images.githubusercontent.com/94519/56149148-2183c780-5f69-11e9-8b42-2b76662e62a3.png">

**Comment by LaurenNiolet** [comment]
thankee

**Comment by LaurenNiolet** [comment]
I don't think I'm blocking this, but in case I am: I approve this message.
